kong/plugins/oauth2/access.lua

```diff
@@ -161,7 +161,7 @@ end
 
 local function get_redirect_uris(client_id)
   local client, err
-  if client_id and client_id ~= "" then
+  if client_id and client_id ~= "" and type(client_id) ~= "boolean" then
```

Comment:
```suggestion
  if type(client_id) == "string" and client_id ~= "" then
```

Reply:
Ah yeah good catch, much cleaner and if ```type(nil)``` check will serve the same purpose as ```if client_id``` check did.

Reply:
Fixed with revisions for this.